src/server/event_bus.c

```
@@ -0,0 +1,265 @@
+
+/* Apply all pending additions accumulated during a fire(). */
+static void flush_additions(obc_event_entry_t *e)
+{
+    for (int a = 0; a < e->add_count; a++)
+        insert_sub(e, e->add_pending[a].fn, e->add_pending[a].priority);
+    e->add_count = 0;
+}
+
+/* --- Public API ----------------------------------------------------------- */
+
+void obc_event_bus_init(void)
```

Cppcheck (2.19.0)

[style] The function 'obc_event_bus_init' is never used.

(unusedFunction)